api/v1beta1/webhook_suite_test.go

```diff
@@ -1,12 +1,9 @@
 /*
-
-
+Copyright 2022.
```

Comment:
I know this is auto-generated, but should we put 2023 here instead?

Reply:
done